docs/sphinx/using/simulators.rst

```diff
@@ -77,101 +98,171 @@ threaded `Q++ <https://github.com/softwareqinc/qpp>`_ library. This is the defau
 target, so if the code is compiled without any :code:`--target` flags, this is the 
 simulator that will be used. 
 
+To execute a program on the :code:`qpp-cpu` target even if a GPU-accelerated backend is available, 
+use the following commands:
+
+.. tab:: C++
+
+    .. code:: bash 
+
+        nvq++ --target qpp-cpu program.cpp [...] -o program.x
+        ./program.x
+
+.. tab:: Python
+
+    .. code:: bash 
+
+        python3 program.py [...] --target qpp-cpu
+
+    The target can also be defined in the application code by calling
+
+    .. code:: python 
+
+        cudaq.set_target('qpp-cpu')
+
+    If a target is set in the application code, this target will override the :code:`--target` command line flag given during program invocation.
+
+
 Tensor Network Simulators
 ==================================
 
 CUDA Quantum provides a couple of tensor-network simulator targets accelerated with 
 the :code:`cuTensorNet` library. 
 These backends are available for use from both C++ and Python.
 
-`cuTensorNet` multi-node multi-GPU
+Tensor network-based simulators are suitable for large-scale simulation of certain classes of quantum circuits involving many qubits beyond the memory limit of state vector based simulators. For example, computing the expectation value of a Hamiltonian via :code:`cudaq::observe` can be performed efficiently, thanks to :code:`cuTensorNet` contraction optimization capability. On the other hand, conditional circuits, i.e., those with mid-circuit measurements or reset, despite being supported by both backends, may result in poor performance. 
+
+Multi-node multi-GPU
 +++++++++++++++++++++++++++++++++++
 
 The :code:`tensornet` backend represents quantum states and circuits as tensor networks in an exact form (no approximation). 
 Measurement samples and expectation values are computed via tensor network contractions. 
-This backend supports Multi-Node, Multi-GPU distribution of tensor operations required to evaluate and simulate the circuit.
+This backend supports multi-node, multi-GPU distribution of tensor operations required to evaluate and simulate the circuit.
 
-.. note:: 
```

Comment:
Move this into a separate section "Enable multi-GPU support" in the Python wheel README, along with moving the MPI install instructions there underneath that section.